File: Practicle-2/second.java
import java.util.*;

class second{
    
    static int Search(int arr[], int n, int key){
       for (int i = 0; i < n; i++)
            if (arr[i] == key)
                return i;
      
        return -1;
    }

    static int insert(int arr[], int n, int key, int cap){
        if (n >= cap)
           return n;
      
        arr[n] = key;
      
        return (n + 1);
    }

    public static void main(String args[]){
        int n = 7;
        Scanner in = new Scanner(System.in);

        int[] arr = new int[10];
        System.out.println("Enter the number of element you want to insert less than 11: ");
        int ch = in.nextInt();
        
        System.out.println("Enter the Element in Array:");
        for(int q=0; q < ch; q++){
            arr[q] = in.nextInt();
        }

        
        //Searching 
        System.out.println("Enter Search Value:");
        int e = in.nextInt();
        int s1 = Search(arr, ch, e);
        System.out.println("Found at: "+ s1);

        //Insertion
        int cap = arr.length;  
        System.out.println("Enter Insert Value:");
        int key = in.nextInt();
        n = insert(arr, ch, key, cap);    
    
        System.out.println("After Insertion: ");
        for(int i=0; i<n; i++){
            System.out.println(arr[i]);
        } 

        //Sorting 
        Arrays.sort(arr);

        System.out.println("After Sorting");
        for(int j=0;j<arr.length;j++){
            System.out.println(arr[j]);
        }
 
    } 
}

File: Practicle-2/second.kt
package Modules

fun main(args: Array<String>){
    println("Enter the Number of Elements in an array")

    var data: Int? = readLine()!!.toInt()
    var arr: ArrayList = ArrayList()
    var search: Int? = null
    for(i in 0 until data!!) {
        arr.add(readLine()!!.toInt())
    }

    println("Enter Number to Search: ")
    search = readLine()!!.toInt()
    println(arr.indexOf(search))

    println("Enter the position to Insert the element: ")
    var pos: Int = readLine()!!.toInt()

    println("Enter the element for Insert: ")
    var ele: Int = readLine()!!.toInt()
    arr.add(pos, ele)

    println("Array is $arr")
    arr.sort()
    println("Sorted Array is $arr")


}
